Generator/source/filesTools/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_library(
+  FilesTools SHARED
+  FilesTools.cpp
+  FilesTools.h
+)
+
+target_include_directories(FilesTools PUBLIC
+${CMAKE_SOURCE_DIR})
```

Comment:
Here GitHub warns about the absence of bank line at the end of the file. Some tools/editors will become nervous too.